konfera/tests/test_models.py

```diff
@@ -94,6 +94,18 @@ def test_slug(self):
                        date_to=date_to, location=location)
         self.assertRaises(IntegrityError, event2.save)  # slug must be unique
 
+    def test_cfp(self):
+        now = datetime.datetime.now()
+        tomorrow = now + datetime.timedelta(days=1)
+        today = datetime.date.today()
+        intwodays = today + datetime.timedelta(days=2)
+        event = Event(title='Test event', date_from=today, date_to=intwodays, cfp_allowed=True)
+        self.assertRaises(ValidationError, event.clean)
+        event1 = Event(title='Test event', date_from=today, date_to=intwodays, cfp_allowed=True, cfp_end=now)
+        self.assertRaises(ValidationError, event1.clean)
+        Event(title='Test normal event', date_from=intwodays, date_to=intwodays, cfp_end=tomorrow)
+        self.assert_(True)
```

Comment:
Ok, I will test it this way. During the tests I found that the abstract FromTo model required date_start to be less then date_end. However, we can have an event that only lasts one day so I changed the logic of validator from ">=" to ">". In case it violates the logic or intention, please let me know so that I can revert it and reset the tests.